Formula/teleconsole.rb

```diff
@@ -29,7 +30,7 @@ class Teleconsole < Formula
 
   go_resource "github.com/gravitational/teleport" do
     url "https://github.com/gravitational/teleport.git",
-        :revision => "002b640a16f097e2f834b4ae33c9edfb81d5798c"
+        :tag => "v2.0.0-alpha.4"
```

Comment:
relates to these lines, https://github.com/gravitational/teleconsole/blob/master/Makefile#L36-L38

Reply:
we should not be shipping an alpha version

Reply:
Yeah, I think that is the stuff in their codebase, maybe I should PR to fix there?

Reply:
@chenrui333 yess please, and also we should be using go modules.

Reply:
Please note, their project does not support `go modules` yet, i.e. the `go.mod` and `go.sum` are not there.

So I think we have to wait until they do the support.